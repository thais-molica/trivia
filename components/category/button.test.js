import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Category from "./index";

it("Category renders correctly", () => {
  const tree = renderer.create(<Category name="category" />).toJSON();
  expect(tree).toMatchSnapshot();
});
