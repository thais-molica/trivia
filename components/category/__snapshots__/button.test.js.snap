// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category renders correctly 1`] = `
<div
  className="jsx-2247176598 box"
  onClick={[Function]}
>
  category
</div>
`;
